examples/debian/aci-debian-cassandra/runlevels/build/10.install.sh

```diff
@@ -10,12 +10,15 @@ deb-src http://www.apache.org/dist/cassandra/debian 30x main
 deb http://ftp.debian.org/debian jessie-backports main
 EOF
 
+gpg --keyserver pool.sks-keyservers.net --recv-keys F758CE318D77295D || \
```

Comment:
Done.

I will issue separate PRs. Once all prerequisites are merged, trim and squash this one.